admin/controllers/restart.js

```diff
@@ -20,11 +24,47 @@ const getSelectRestartList = async (req, res, next) => {
   }
   return res.render('restart/select-restart-list', {
     breadcrumbs: req.breadcrumbs(),
-    pupils
+    pupils,
+    error: new ValidationError()
   })
 }
 
-module.exports = {
-  getRestartOverview,
-  getSelectRestartList
+controller.postSubmitRestartList = async (req, res, next) => {
+  const { pupil: pupilsList, restartReason, didNotCompleteInfo, restartFurtherInfo } = req.body
+  if (!pupilsList) {
+    return res.redirect('/restart/select-restart-list')
+  }
+  const validationError = restartValidator.validateReason(restartReason, didNotCompleteInfo)
+  if (validationError.hasError()) {
+    res.locals.pageTitle = 'Select pupils for restart'
```

Comment:
'Error: ' needs to be a prefix of the page title - we've been missing this AC but should start implementing it ASAP.
"Additionally, you should reflect that there's been an error in the <title> by prefixing the existing title with 'Error: '".
https://govuk-elements.herokuapp.com/errors/

Reply:
The problem with GDS here is that the don't really provide an example on this.
Do we need to prefix the page title or the error title?